utils/rpms/daos.spec

```diff
@@ -256,6 +256,7 @@ getent passwd daos_agent >/dev/null || useradd -s /sbin/nologin -r daos_agent
 # you might think libdaos_tests.so goes in the tests RPM but
 # the 4 tools following it need it
 %{_libdir}/libdaos_tests.so
+%{_sysconfdir}/ld.so.conf.d/daos.conf
```

Comment:
So you will want to remove this from the server package now:
https://github.com/daos-stack/daos/blob/f4c76a5e02f6c8d18aa1136a207311f85aa1f298/utils/rpms/daos.spec#L289

Since the package it's now in is _Required_ by the server package.